### ISSUE TITLE
Force H2 Retrofit of German Gas Plants

### ISSUE THREAD

**Issue by toniseibold** [PyPSA/pypsa-ariadne#104]
Once [PR](https://github.com/PyPSA/pypsa-eur/pull/1035) is merged into `ariadne2`, all gas plants (OCGT, CCGT and CHP) need to be forced to switch to hydrogen from a certain year on:
[Kraftwerksstrategie](https://www.bundesregierung.de/breg-de/aktuelles/kraftwerksstrategie-2257868) currently expects a switch in 2035-2040.
